modules/nw-using-service-external-ip.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * ingress/getting-traffic-cluster.adoc
+
+[id="nw-service-external-ip-{context}"]
+= Using a service external IP to get traffic into the cluster
+
+One method to expose a service is to assign an external IP access directly to
+the service you want to make accessible from outside the cluster.
+
+Make sure you have created a range of IP addresses to use, as shown in Defining
+the Public IP Address Range.
+
+By setting an external IP on the service, OpenShift Container Platform sets up
+IP table rules to allow traffic arriving at any cluster node that is targeting
```

Comment:
We're moving from iptables to nftables, but it's probably best to use abstract language here; e.g., "sets up NAT rules".